.promu.yml

```diff
@@ -5,7 +5,11 @@ go:
 repository:
     path: github.com/prometheus/blackbox_exporter
 build:
-    flags: -a -tags netgo
+    tags:
+        all:
+            - netgo
+        windows:
+    flags: -a
```

Comment:
This is actually an obsolete param setup. It can be fully removed as the defaults work correctly now.

```suggestion
```